src/main/java/ParkingLot/VehicleParking.java

```diff
@@ -0,0 +1,109 @@
+package ParkingLot;
+
+import java.util.*;
+
+public class VehicleParking {
+
+
+    final static int LOT_COUNT = 100;
+    static int freeLots = 100;
+    static String pattern = "([А-Я][А-Я](\\s)[0-9][0-9][0-9][0-9](\\s)[А-Я][А-Я])";
+    static List<String> lots = new ArrayList();
+    private static List<String> uniqueNumbers = new ArrayList();
+    static List<Vehicle> carAll = new ArrayList();
+    static Map<String, Vehicle> carNumLotMap = new HashMap<>();
+
+    static void printVehicleList(List list) {
+
+        list.stream().forEach(System.out::println);
+    }
+
+    static void filterByNumber(String template) {
+        for (Vehicle vehicle : carAll) {
+            if (vehicle.getNumber().substring(0, 2).equals(template)) {
+                System.out.println(vehicle);
+            }
+        }
+    }
+
+    static boolean checkIfExistNumber(String template) {
+        for (Vehicle vehicle : carAll) {
+            if (vehicle.getNumber().substring(0, 2).equals(template)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    static void waitConsole() {
+        try {
+            Thread.sleep(1000);
```

Comment:
i'm wondering, what is the motivation for thread sleep?

Reply:
if i use it, i can have different checkIN time for Vehicle objects, and sort them by duration.